### PR TITLE
Added disk util percentage to load endpoint

### PR DESCRIPTION
**Pull request by alexwang08**
Added disk util in percentage to load endpoint. Also corrected presentation of CPU util to be "percent". Also made minor java doc fix.

### CODE REVIEW

File: cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/stats/BasicStats.java [hunk 1]
```diff
@@ -89,6 +124,7 @@ void addBasicStats(BasicStats basicStats) {
   public Map<String, Object> getJSONStructure() {
     Map<String, Object> entry = new HashMap<>(8);
```
Comment (efeg) [inline cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/stats/BasicStats.java]: Let's update the initial capacity: `new HashMap<>(8)` -> `new HashMap<>(9)`

File: cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/ClusterModel.java [hunk 2]
```diff
@@ -1004,7 +1004,7 @@ public BrokerStats brokerStats() {
                                        broker.load().expectedUtilizationFor(Resource.NW_OUT),
                                        potentialLeadershipLoadFor(broker.id()).expectedUtilizationFor(Resource.NW_OUT),
                                        broker.replicas().size(), broker.leaderReplicas().size(),
-                                       _capacityEstimationInfoByBrokerId.get(broker.id()) != null);
+                                       _capacityEstimationInfoByBrokerId.get(broker.id()) != null, broker.capacityForAll());
```
[the remark this answers is not in the document]
Comment (alexwang08) [reply cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/ClusterModel.java]: was trying to make it extensible so if we need to calculate utilization for other resources, we don't have to add another parameter to pass in.

File: cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/stats/BasicStats.java [hunk 3]
```diff
@@ -25,24 +28,43 @@
   private double _potentialBytesOutRate;
   private int _numReplicas;
   private int _numLeaders;
+  private double[] _brokerCapacity;
```
[the remark this answers is not in the document]
Comment (alexwang08) [reply cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/stats/BasicStats.java]: was trying make capacity more general so as to not limit to diskCapacity

File: cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/Broker.java [hunk 4]
```diff
@@ -116,6 +117,19 @@ public double capacityFor(Resource resource) {
     return -1.0;
   }
 
+  /**
+   * Get broker capacity for all resources
+   * @return If broker is alive, an array of the capacity of the requested resources, an array of -1.0 otherwise.
+   */
+  public double[] capacityForAll() {
```
Comment (efeg) [inline cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/Broker.java]: Shall we remove this unused function?

File: cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/stats/BasicStats.java [hunk 5]
```diff
@@ -25,24 +26,36 @@
...
     return _diskUtil;
   }
 
+  // Return -1 if total disk space is invalid. Since unit is in percent, will return the digits without
+  // percent sign. e.g. return 99.9 for 99.9%
+  double diskUtilPct() {
+    if (_diskCapacity > 0) {
+      return 100 * _diskUtil / _diskCapacity;
+    }
+    return -1.0;
```
Comment (efeg) [inline cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/stats/BasicStats.java]: Nit: `return _diskCapacity > 0 ? 100 * _diskUtil / _diskCapacity : -1.0;`?

File: cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/stats/BasicStats.java [hunk 6]
```diff
@@ -71,6 +84,10 @@ int numLeaders() {
     return _numLeaders;
   }
 
+  double getDiskCapacity() {
```
Comment (efeg) [inline cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/stats/BasicStats.java]: Nit: just a minor convention for `getters` we try to use in cc: `getDiskCapacity()` -> `diskCapacity()`

**Pull request merged**